src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -8,11 +10,122 @@
 from sentry import features
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
-from sentry.api.paginator import OffsetPaginator
-from sentry.api.serializers import serialize
+from sentry.api.paginator import OffsetPaginator, SequencePaginator
+from sentry.api.serializers import Serializer, register, serialize, RuleSerializer
+from sentry.api.serializers.models.rule import _generate_rule_label
 from sentry.incidents.models import AlertRule
+from sentry.models import Rule, RuleStatus
 from sentry.incidents.endpoints.serializers import AlertRuleSerializer
 
+class CombinedRuleSerializer(Serializer):
+    def serialize(self, obj, attrs, user, **kwargs):
+        rule = obj
+        # TODO: Use native serializer instead of cut/paste
```

Comment:
When you do this, you'll likely want to do it in `get_attrs` so that you can serialize in bulk